### ISSUE TITLE
[wg/rdf-star] all errata should be in scope

### ISSUE THREAD

**Issue by pchampin** [w3c/charter-drafts#615]
Two members suggested changes, while supporting the proposal whether or not the changes are adopted :

> [Member A] supports this work but suggests a change in the scope section of the charter. The section states "For every recommendation updated by this Working Group, the pending editorial errata will also be addressed." Given the extended time provided to the group by the new charter, it is worth extending the scope to all kinds of errata, including the substantive ones. There is little point in publishing new recommendations with unaddressed errata, some of which being a decade old. [ Member A] believes such errata should be in the scope.

And, from another member:

> We support the changes suggested by [Member A].

**Comment by afs** [comment]
Minor: the "RDF 1.2 XML Syntax" item still says "editorial".

**Comment by pchampin** [comment]
> Minor: the "RDF 1.2 XML Syntax" item still says "editorial".

yes, this was also intended. Already at the time of RDF 1.1, there was not much interest in the WG to update RDF/XML beyond editorial errata. Assuming there would not be much more interest today, this mention was to avoid raising expectations too much -- but it still opens the door to more substantive changes if people are motivated.

**Comment by plehegar** [comment]
https://www.w3.org/2025/04/rdf-star-council-report.html
